pkg/config/v2/upstream.go

```diff
@@ -121,6 +123,13 @@ type DnsResolverConfig struct {
 	Attempts int      `json:"attempts,omitempty"`
 }
 
+type SlowStartConfig struct {
+	Mode              string              `json:"mode,omitempty"`
+	SlowStartDuration *api.DurationConfig `json:"slow_start_duration,omitempty"`
+	Aggression        float64             `json:"aggression,omitempty"`
+	MinWeightPercent  *Percent            `json:"min_weight_percent,omitempty"`
```

Comment:
Why not use `float64` type directly?

Reply:
As same as envoy `Percent`, I will fix it